packages/inference/src/lib/makeRequestOptions.ts

```diff
@@ -89,6 +90,9 @@ export async function makeRequestOptions(
 			provider === "fal-ai" && authMethod === "provider-key" ? `Key ${accessToken}` : `Bearer ${accessToken}`;
 	}
 
+	// e.g. @huggingface/inference@3.1.3
+	headers["User-Agent"] = `${packageName}@${packageVersion}`;
```

Comment:
```suggestion
	headers["User-Agent"] = `${packageName}/${packageVersion}`;
```

To harmonize with `huggingface_hub` cc @Wauplin @hanouticelina

Reply:
for the package you can just use hardcoded `huggingface.js/{version_number}` IMO (potentially this would apply to all calls from all hf.js subpackages, no need to special-case inference)

Reply:
we have different versions for different sub-pacakges though :(

maybe `huggingface.js;@huggingface/{subPackage}/{version}`